libclamav/unzip.c

```diff
@@ -624,6 +624,9 @@ static unsigned int parse_local_file_header(
     uint32_t csize, usize;
     unsigned int size_of_fileheader_and_data = 0;
 
+    uint32_t nsize;
```

Comment:
Would suggest initializing these to 0/NULL for a little future-proofing.

Reply:
Fixed